docs/deployment/choosing_configuring_rds.md

```diff
@@ -77,39 +83,10 @@ Current security policy/best practices require use of a SSL enabled configuratio
 
 Cumulus can accommodate a self-signed/unrecognized cert by setting `rejectUnauthorized` as `false` in the connection secret.  This will result Core allowing use of certs without a valid CA.
 
-## Recommended Scaling Configuration for Aurora Serverless
```

Comment:
This isn't applicable in V2. The most relevant configuration options here are the min and max capacity settings. We _could_ provide some guidance there but I don't believe we have that ready to go. Totally open to thoughts here as this was the most contentious part of these doc updates in my mind.